src/core/components/keep-alive.js

```diff
@@ -120,5 +120,13 @@ export default {
       vnode.data.keepAlive = true
     }
     return vnode || (slot && slot[0])
+  },
+  updated() {
+    const { cache } = this
```

Comment:
Hello @dejour , just passing by and noticed that you can also destructure the _vnode property in this statemant, avoiding to use `this._vnode` on the line 127.

Also, the CI points a build failure on the test-cover and lint steps. I think it has nothing to do with your code, since the change it's not on you commit.